simple_database/main.py

```diff
@@ -1,5 +1,5 @@
-def create_database(db_name):
-    raise NotImplementedError()
+# def create_database(db_name):
```

Comment:
Why not eliminate the effectively empty file?  or move your sub file into this one, if you are exposing your user functions via dbmodule.py?